client/src/components/section3/Section3.jsx

```diff
@@ -1,17 +1,128 @@
 import React, { Component } from 'react';
+import styled from 'styled-components';
+import { ChevronDown, ChevronUp } from 'react-feather';
+import Modal from './Modal.jsx';
+
 
 class Section3 extends Component {
     constructor(props) {
-        super(props)
-    }
+        super(props);
+
+
+        this.state = {
+            status: 'collapse',
+            show: false,
+            idx:  Math.floor(Math.random() * 100)
```

Comment:
Is there a reason you put this here instead of as a variable in the render function? @dekim2324